spec/grape_entity/entity_spec.rb

```diff
@@ -296,24 +296,13 @@ class Parent < Person
           expect(subject.exposures).to eq(name: {}, email: {})
         end
 
-        # the following 2 behaviors are testing because it is not most intuitive and could be confusing
-        context 'when  called from the parent class' do
-          it 'remove from parent and all child classes that have not locked down their attributes with an .exposures call' do
+        context 'when called from the parent class' do
+          it 'remove from parent and do not remove from child classes' do
             subject.expose :name, :email
             child_class = Class.new(subject)
             subject.unexpose :email
 
             expect(subject.exposures).to eq(name: {})
-            expect(child_class.exposures).to eq(name: {})
-          end
-
-          it 'remove from parent and do not remove from child classes that have locked down their attributes with an .exposures call' do
```

Comment:
Because there is no more "confusing" behaviour. `unexpose` no longer affects child classes regardless of was `.exposures` method called in them or not.